src/main/java/org/fcrepo/spec/testsuite/AbstractTest.java

```diff
@@ -261,21 +289,37 @@ protected Response doOptions(final String uri) {
         return response;
     }
 
-
     private RequestSpecification createRequest() {
-        return createRequestAuthOnly().config(RestAssured.config().redirect(redirectConfig().followRedirects(false))
-                                      .logConfig(new LogConfig().defaultStream(ps)
-                                                                .enableLoggingOfRequestAndResponseIfValidationFails()))
-                                      .log().all();
+        return createRequest(true);
+    }
+
+    private RequestSpecification createRequest(final boolean admin) {
+        return createRequestAuthOnly(admin)
+            .config(RestAssured.config().redirect(redirectConfig().followRedirects(false))
+                               .logConfig(new LogConfig().defaultStream(ps)
+                                                         .enableLoggingOfRequestAndResponseIfValidationFails()))
+            .log().all();
     }
 
     private RequestSpecification createRequest(final String slug, final String contentType) {
         return createRequest().header(SLUG, slug).contentType(contentType);
     }
 
     private RequestSpecification createRequestAuthOnly() {
```

Comment:
This private method is no longer used, and can be removed.

Reply:
It is used in several places in AbstractTest.java